src/org/parosproxy/paros/extension/manualrequest/http/impl/HttpPanelSender.java

```diff
@@ -311,4 +322,17 @@ public URI getInvalidRedirection() {
             return invalidRedirection;
         }
     }
+    
+    private void setUseTrackingSessionState(boolean shouldUseTrackingSessionState) {
+        if (!shouldUseTrackingSessionState) {
+            if (delegate != null) {
+                delegate.setUseGlobalState(shouldUseTrackingSessionState);
+            }
+        }
+        getButtonUseTrackingSessionState().setSelected(shouldUseTrackingSessionState);

```

Comment:
This should be moved to `setButtonTrackingSessionStateEnabled` (done when `enabled` is `false`).